packages/benchmark/src/benchmarkerator.js

```diff
@@ -586,4 +592,5 @@ export const makeBenchmarkerator = async () => {
 /**
  * The normal singleton benchmarkerator.
  */
+// eslint-disable-next-line @jessie.js/safe-await-separator
 export const bench = await makeBenchmarkerator();
```

Comment:
The current state of Benchmarkerator seems far too anchored to agoric blockchain details (vat configuration, smart wallets, etc.) to be associated with such a generic name IMO.

Reply:
I agree for the most part, but claiming the generic name now therefore feels premature. I think it would be better to start with something like `benchmarkEconFlow` and expect renaming as things evolve.